tests/infra/runner.py

```diff
@@ -148,11 +148,18 @@ def run(get_command, args):
 
                     time.sleep(5)
 
+                perf_label = args.perf_label
+                if infra.platform_detection.is_virtual():
+                    perf_label += "_virtual"
```

Comment:
Doing this at runtime, we may want something more granular like "_genoa4c16g" etc, but for now this is fine and keeps continuous time series